src/components/Profile/Profile/FeatureContainer/UserVisits/UserVisit.tsx

```diff
@@ -1,14 +1,24 @@
+import { ROUTS } from '@constants/routs';
+import { useTypesSelector } from '@hooks/UseTypedSelector';
 import React from 'react';
-import { ContainerName, VisitContainer } from './UserVisitStyle';
+import { NavLink } from 'react-router-dom';
+import { ContainerName, VisitContainer, ErrorMessage } from './UserVisitStyle';
 import VisitItem from './VisitItem.tsx/VisitItem';
 
 const UserVisit: React.FC = () => {
+  const { visit } = useTypesSelector((state) => state.profile.profileData);
+
   return (
     <VisitContainer>
       <ContainerName>My visit</ContainerName>
-      <VisitItem />
-      <VisitItem />
-      <VisitItem />
+      {!visit || visit.length === 0 ? (
```

Comment:
what's the point of checking length to be equal to 0? It will be false even without checking it's value

Reply:
![image](https://user-images.githubusercontent.com/56519328/141693709-3e51e599-261f-408e-9a04-513d441ca8ea.png)
![image](https://user-images.githubusercontent.com/56519328/141693714-1da5b7d2-f6d9-46c6-ad5a-d68f11c78e50.png)

I don't understand why then the span is not displayed

Reply:
what span is not displayed?

Reply:
fixed